index.js

```diff
@@ -942,7 +977,7 @@
   //. > S.init([])
   //. Nothing()
   //. ```
-  S.init = R.ifElse(R.isEmpty, Nothing, R.compose(Just, R.init));
+  S.init = S.slice(0, -1);
```

Comment:
Let's accept `-0` to indicate the position half a step beyond the last index. Currently we have:

```javascript
 [ 'a' , 'b' , 'c' , 'd' , 'e' ]

 0     1     2     3     4     5
-5    -4    -3    -2    -1
```

Instead we could have:

```javascript
 [ 'a' , 'b' , 'c' , 'd' , 'e' ]

 0     1     2     3     4     5
-5    -4    -3    -2    -1    -0
```

This would facilitate this symmetry:

```javascript
S.tail = S.slice(1, -0);
S.init = S.slice(0, -1);
```

With `R.slice` one can use `Infinity` as the end index, but that does not work when the index needs to be within the array's bounds.

The most straightforward expression to detect `-0` is `R.toString(x) === '-0'`.